pkg/kuberecord/recorder_test.go

```diff
@@ -34,7 +35,11 @@ func TestRecorder(t *testing.T) {
 
 	cfg, err := env.Start()
 	require.NoError(t, err)
-	defer env.Stop()
+	defer func(env *envtest.Environment) {
+		if err := env.Stop(); err != nil {
+			log.Error()
```

Comment:
Can we `t.Logf` here to avoid prometheus?

Reply:
Done